reference/employer/openapi.yaml

```diff
@@ -926,6 +932,288 @@ paths:
       description: Disconnect an employer from your application and invalidate all `access_token`s associated with the employer. We require applications to implement the Disconnect endpoint for billing and security purposes.
       tags:
         - Management
+  /employer/company/benefits:
+    get:
+      summary: List Company Benefits
+      tags:
+        - Employer
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  type: object
+                  properties:
+                    benefit_id:
+                      type: string
+                    type:
+                      type: string
+                    description:
+                      type: string
+                    frequency:
+                      type: string
+                    company_contribution:
+                      type: string
+                    employee_deduction:
+                      type: string
+      operationId: get-company-benefits
+      description: List all company-wide benefits.
+    post:
+      summary: Create Company-wide Benefit
+      operationId: post-company-benefits
+      responses:
+        '201':
+          description: Successfully Created
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  benefit_id:
+                    type: string
+                  description:
+                    type: string
+      description: Creates a new company-wide benefit
+      requestBody:
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                type:
+                  type: string
+                description:
+                  type: string
+                employee_deduction:
+                  type: object
+                  properties:
+                    type:
+                      type: string
+                    amount:
+                      type: number
```

Comment:
I guess something we didn't chat about was the format of this number. Is this cents only? If so - should this be an integer?